doc/source/data_load/spark.md

```diff
@@ -1 +1,210 @@
-# Apache Spark
+## Apache Spark
+
+Spark is a unified analysis engine for large-scale data processing. Hologres and Saprk (Community Edition and EMR Spark Edition) are efficiently opened up, providing Spark Connector, supporting data from Spark to be written into Hologres in stream or batch mode, and quickly helping enterprises to build data warehouses.
+
+## Supported Spark Versions
+
+2.4.3 or 2.4.3+
+
+## 1. JDBC Connector
+
+### Batch Writing
+
+Use spark jdbc connector directly, one example below, download postgres jdbc jar file
+
+```
+./bin/spark-shell --jars postgresql-42.2.14.jar
+```
+
+```
+import org.apache.spark.sql.types._
+import org.apache.spark.sql.Row
+
+val data = Seq(
+    Row(1L, "test"),
+    Row(2L, "test2"),
+    Row(3L, "test2"),
+    Row(4L, "test2"),
+    Row(5L, "test2"),
+    Row(6L, "test2")
+)
+val schema = StructType(Array(
+ StructField("a", LongType),
+ StructField("b", StringType)
+))
+
+val df = spark.createDataFrame(spark.sparkContext.parallelize(data),schema)
+
+// .option("checkpointLocation", checkpointLocation)
+df.write.format("jdbc")
+  .option("url", "jdbc:postgresql://hgcn-xxx.yyy.zzz:80/db001")
+  .option("driver", "org.postgresql.Driver")
+  .option("dbtable", "tb001")
+  .option("user", "Lxxxxxxxx")
+  .option("password", "Dyyyyyyyyyyyyyyyyyyyy")
+  .option("isolationLevel", "NONE")
```

Comment:
what does 'isolationLevel' mean?

Reply:
We do not support isolation level it will be deleted